src/OctoshiftCLI.Tests/gei.Commands/EnvironmentVariableProviderTests.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using FluentAssertions;
+using Moq;
+using OctoshiftCLI.GithubEnterpriseImporter;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.GithubEnterpriseImporter.Commands
+{
+    public class EnvironmentVariableProviderTests
+    {
+        private const string SOURCE_GH_PAT = "SOURCE_GH_PAT";
+        private const string TARGET_GH_PAT = "TARGET_GH_PAT";
+
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        public EnvironmentVariableProviderTests()
+        {
+            _environmentVariableProvider = new EnvironmentVariableProvider(new Mock<OctoLogger>().Object);
+        }
+
+        [Fact]
+        public void SourceGithubPersonalAccessToken_Should_Return_Github_Source_Pat()
+        {
+            // Arrange
+            Environment.SetEnvironmentVariable("GH_SOURCE_PAT", SOURCE_GH_PAT);
```

Comment:
NOTE: These tests are not going to run in parallel because by default they are in the same xUnit collection and this is going to be the only place where we set the env variables. So it should be ok to set them. For the rest of the tests we always mock the `EnvironmentVariableProvider`